www/app/Controller/Page/Minigame.php

```diff
@@ -4,16 +4,31 @@
 
 use App\Controller\LoggedIn;
 use App\Controller\Response;
+use App\Model\User;
+use App\Util\Singleton\Session;
 use App\Util\ViewRenderer;
 
 /**
  * Minigame "Thijs zijn nachtmerrie".
  */
 class Minigame implements Response, LoggedIn
 {
+    public function __construct()
+    {
+        if (isset($_POST['highscore'])) {
+            $user = Session::instance()->getUser()->updateHighscore($_POST['highscore']);
+            Session::instance()->setUser($user);
+        }
```

Comment:
Dit kan natuurlijk gecheat worden maar ik betwijfel of daar iets aan te doen is.

Reply:
Misschien moeten beheerders een highscore kunnen resetten. Maar geen prioriteit.